### DISCUSSION TITLE
getListing is not returning shop information

### DISCUSSION THREAD

**Discussion by kat0310** [etsy/open-api#1189] [Q&A] [answered]
Hi,
according to the Etsy v3 documentation getListing should also return shop information. Whan making a call I only get a response for "shop": None

any ideas?

**Comment by showtroylove** [comment, marked as answer]
It only returns Shop information when you supply the optional **includes** string list namely, "[Shop](https://developers.etsy.com/documentation/reference/#operation/getListing)".  

If this information helps you please press the resolved / answered button.

**Reply by kat0310** [reply]
Thank you so much. I thought that i checked the complete documentation but there was one point left that I haven't ;)
It works now
